examples/nextjs-blog/pages/blog/post.js

```diff
@@ -1,153 +1,208 @@
-import React, { PureComponent } from 'react'
-import Markdown from 'markdown-to-jsx'
+import React, { useState, useEffect } from 'react'
 import PropTypes from 'prop-types'
+import get from 'lodash/get'
+import Markdown from 'markdown-to-jsx'
 import NextError from 'next/error'
+import NextSeo, { ArticleJsonLd } from 'next-seo'
 import { flamelinkApp as app } from '../../utils/flamelink'
-import { getImageAlt, getDateString } from '../../utils/post/post.util'
+import {
+  getImageAlt,
+  getDateString,
+  getDateFromTimestamp,
+} from '../../utils/post/post.util'
 import { DEFAULT_POST_IMAGE_URL } from '../../constants/constants'
 import { containerInner } from '../../components/styled'
 
-class Post extends PureComponent {
-  constructor(props) {
-    super(props)
-    this._subscription = null
-    this.state = {
-      post: null,
-    }
-  }
+const Post = props => {
+  const [error, setError] = useState(null)
+  const [updatedPost, setPost] = useState(null)
 
-  async componentDidMount() {
+  useEffect(() => {
     const { pathname } = location
```

Comment:
PR to clean these up (PR is to merge into your branch)

Reply:
Thanks d-dubs